go/vt/vtgate/engine/filter_test.go

```diff
@@ -96,19 +120,11 @@ func TestFilterMixedFail(t *testing.T) {
 	for _, tc := range tcases {
 		t.Run(tc.name, func(t *testing.T) {
 			filter := &Filter{
-				Predicate: predicate,
+				Predicate: pred,
 				Input:     &fakePrimitive{results: []*sqltypes.Result{tc.res}},
 			}
 			_, err := filter.TryExecute(context.Background(), &noopVCursor{}, nil, false)
 			require.EqualError(t, err, tc.expErr)
 		})
 	}
 }
-
-func defaultCollation() collations.TypedCollation {
-	return collations.TypedCollation{
-		Collation:    collationEnv.LookupByName("utf8mb4_bin").ID(),
```

Comment:
I guess it was moved from here.

Reply:
That's right, the test was using the `utf8mb4_bin` collation so we've just ported that.